### ISSUE TITLE
Installing mscgenjs-cli 1.3.5 in node4/ npm2 fails with a phantomjs-prebuilt installation error

### ISSUE THREAD

**Issue by sverweij** [mscgenjs/mscgenjs-cli#12]
- When installing mscgenjs-cli@1.3.5 in nodejs the system produces the error + trace as below
- In nodejs5/ npm3: no probs.

```
$ npm i -g mscgenjs-cli --production

> phantomjs-prebuilt@2.1.4 install /Users/sverweij/.nvm/versions/node/v4.3.1/lib/node_modules/mscgenjs-cli/node_modules/phantomjs-prebuilt
> node install.js

module.js:327
    throw err;
    ^

Error: Cannot find module './util/assign'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/sverweij/.nvm/versions/node/v4.3.1/lib/node_modules/mscgenjs-cli/node_modules/fs-extra/lib/index.js:1:76)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
```

**Comment by sverweij** [comment]
- Apparently the npm-shrinkwrap created with npm3 does not play well in npm2. 
- Removing the npm-shrinkwrap.json from the distribution solves the problem (confirmed locally and on travis _with clean caches_)

Fixed by 257bc66f4f865

**Comment by sverweij** [comment]
Fix published to npmjs.com - version [1.3.6](https://github.com/sverweij/mscgenjs-cli/releases/tag/1.3.6)
